photons_control/transform.py

```diff
@@ -127,10 +132,19 @@ def receiver(serial, current_state):
             currently_off = current_state.power == 0
 
             if currently_off:
+
                 clone = color_message.clone()
                 clone.period = 0
                 clone.brightness = 0
                 clone.set_brightness = True
+
+                clone.hue = 0 if transition_color else clone.hue
```

Comment:
you'll want to keep hsbk the new colors if transition_color is false rather than resetting them

Reply:
oh, you are. But we can still just leave them alone because the message says ignore them

Reply:
It doesn't make any difference. The initial packet has `set_hue` and `set_saturation` and `set_kelvin` to 0 anyway.

Reply:
oh it won't make a difference, but setting hsk to 0 is unecessary because the message says to ignore them if transition_color is true

Reply:
The only reason I added this is because I didn't want to go and change all the other tests because the only way I could get `set_hue` and `set_saturation` to `0` was to override them with `None` values and I needed this initial packet to match.

Reply:
oh yeah, you could do it like

```
msg = Parser.color_to_msg("blue", overrides={"brightness": 0}
msg.set_hue = 0
msg.set_saturation = 0
msg.set_kelvin = 0
```

and use that msg in your expectation

Reply:
Ah, that's more elegant. Let me push that change.